### ISSUE TITLE
Script support?

### ISSUE THREAD

**Issue by chchrist** [tkadlec/grunt-perfbudget#12]
Hello,
I haven't checked the code (lazy me) but is there support for scripts?
I see that you use https://github.com/marcelduran/webpagetest-api
So instead a url can I pass a script?

**Comment by travi** [comment]
Any plan to add this functionality?

**Comment by tkadlec** [comment]
Yeah, I really want to get this baked in. Lots of good uses, I think.

**Comment by cweekly** [comment]
+1 for script support; short of a PR, anything I can do to help?
